js/util/templateHelpers.js

```diff
@@ -36,6 +36,91 @@ define(["globals", "lib/faces", "lib/knockout", "util/helpers"], function (g, fa
         }
     };
 
+    ko.bindingHandlers.letterGrade = {
+      update: function(element, valueAccessor) {
+        var arg, output;
+        arg = ko.unwrap(valueAccessor());
+
+        if (arg < 0.20) {
```

Comment:
I will just submit one final plea that while the toggle view may appear heavy in the screen-shot, in practice it feels much lighter and easier to comprehend than that base ratings. (or this hybrid view)  I've pretty much gotten myself hooked on the full composite view, it just makes getting a read on a player orders of magnitudes easier from my experience so far.